### ISSUE TITLE
Examples too general

### ISSUE THREAD

**Issue by UGME** [MicrosoftDocs/dynamics365smb-docs#247]
The 3 steps don't lead to a conclusion.  Yes you can create the Categories - would have been nice to include the navigation steps or a link to a doc/video.  Then there is the need for a chart to be built (silent on that part).

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 96d2116f-83ee-acf5-48f8-73abc4a00287
* Version Independent ID: 3f257051-6e66-67ef-40c9-add293f6b2e4
* Content: [Learn About General Ledger and COA - Business Central](https://docs.microsoft.com/en-us/dynamics365/business-central/finance-general-ledger#feedback)
* Content Source: [business-central/finance-general-ledger.md](https://github.com/MicrosoftDocs/dynamics365smb-docs/blob/live/business-central/finance-general-ledger.md)
* Service: **dynamics365-business-central**
* GitHub Login: @edupont04
* Microsoft Alias: **edupont**

**Comment by rubenseishima** [comment]
Hi,

Sorry for taking this long to reply. We have updated the article and it will be published in the near future.
Regarding the Chart of Accounts, since this issue was first published, the [Set Up or Change the Chart of Accounts](https://learn.microsoft.com/dynamics365/business-central/finance-setup-chart-accounts) article was updated to include video instructions and more details on available options.
